README.md

```diff
@@ -53,54 +26,139 @@ The source for these tags is at [_plugins/links.rb](_plugins/links.rb).
 | ghc   | GitHub code link          | Branch defaults to `gh_branch` setting in `_config.yml`. Override using `-b` | `{% ghc server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java %}` `{% ghc -b 1.9 README.md %}` |
 | jira   | Jira issue link          | None  | `{% jira ACCUMULO-1000 %}` |
 
-## Updating property documentation
+### Updating Property Documentation for Releases
 
-Building Accumulo  generates `server-properties.md` and `client-properties.md`.  To
-regenerate these, do the following.
+Building Accumulo  generates `server-properties.md` and `client-properties.md`.
+To regenerate these, do the following.
 
-```
-cd <accumulo source dir>
+```bash
+ACCUMULO_SITE_CLONE=<accumulo website clone location, with master branch checked out>
+ACCUMULO_CLONE=<accumulo clone location>
+cd "$ACCUMULO_CLONE"
 mvn package -DskipTests
-cp ./core/target/generated-docs/server-properties.md <accumulo website source>/_docs-2/configuration
-cp ./core/target/generated-docs/client-properties.md <accumulo website source>/_docs-2/configuration
+cp ./core/target/generated-docs/server-properties.md "$ACCUMULO_SITE_CLONE"/_docs-2/configuration
+cp ./core/target/generated-docs/client-properties.md "$ACCUMULO_SITE_CLONE"/_docs-2/configuration
 ```
 
-## Update the production website
+## Local Builds for Testing
+
+### Setting up Your Ruby/Bundler Environment
+
+Ruby and RubyGems are required to use Jekyll and Bundler, so first make sure
+you have those on your machine.
```

Comment:
It sounds like Ruby and RubyGems are a prerequisite to these instructions instead of the goal. I think something like this would be better:
"Ruby and RubyGems are required to use Jekyll and Bundler.  Follow these guidelines to install each of the pieces on your machine."

Reply:
Ruby and RubyGems **are** a prerequisite. The next paragraph talks about the `-devel` packages and other tools that you might also need, but the instructions do not cover installing ruby or rubygems themselves.

Reply:
I updated the section header to clarify that the goal is to install the prerequisites for Jekyll/Bundler.